app/views/practices/show/overview/_overview_sections.html.erb

```diff
@@ -80,7 +80,9 @@
           <% pr.link_url = "http://" + pr.link_url %>
         <% end %>
         <div class="margin-top-1 width-full line-height-26">
-          <li class="small-disc"><%= pr.description %> <a class="usa-link <%= pr.link_url.downcase().include?('marketplace.gov') ? '' : 'usa-link--external' %>" href="<%= pr.link_url %>" target="_blank"><%= pr.name %></a></li>
+          <li class="small-disc"><%= pr.description %>
+              <a class="usa-link <%= @allowed_hosts.include?(pr.link_url) ? '' : 'usa-link--external' %>" <%= @allowed_hosts.include?(pr.link_url) ? '' : 'target="_blank" ' %> href="<%= pr.link_url %>"> <%= pr.name %></a>
+          </li>
```

Comment:
I think we can just refactor what we had, changing 
```
pr.link_url.downcase().include?('marketplace.gov')
```
to
```
pr.link_url.downcase().include?('marketplace.va.gov')
```

Reply:
Thats what I originally had... but I get this security warning in git hub with that code.. 
![image](https://user-images.githubusercontent.com/60527788/174191995-03331c12-bd82-488d-9043-ae7adac800c4.png)
We can discuss tomorrow.